code/obj/item/kitchen.dm

```diff
@@ -887,11 +887,14 @@ TRAYS
 						var/image/rolltopping = new /image('icons/obj/kitchen.dmi',"roll_topping-[i]")
 						switch(i)
 							if(1)
-								rolltopping.color = topping1.food_color
+								if(topping1)
```

Comment:
Honestly the toppings should probably just be a list.

Reply:
Probably...I did it this way to potentially save memory. I'm not sure which is more intensive for DM. Also I have a hunch that the issue may have something to do with setting a variable to a reference to an object, then subsequently qdeling that object which is nulling the reference. Which would explain why the bug seems server dependent. This is a temporary fix to get the mechanic to stop fully breaking, but I'll see if i can fix it further soon.